src/kernel-install/kernel-install.in

```diff
@@ -217,15 +217,15 @@ for pref in $BOOT_ROOT_SEARCH; do
         else
             log_verbose "$pref/$suff not found…"
         fi
-
-        if [ -d "$pref/loader/entries" ]; then
-            [ -z "$BOOT_ROOT" ] && BOOT_ROOT="$pref"
-            log_verbose "$pref/loader/entries exists, using BOOT_ROOT=$BOOT_ROOT"
-            break 2
-        else
-            log_verbose "$pref/loader/entries not found…"
-        fi
     done
+
+    if [ -d "$pref/loader/entries" ]; then
+        [ -z "$BOOT_ROOT" ] && BOOT_ROOT="$pref"
+        log_verbose "$pref/loader/entries exists, using BOOT_ROOT=$BOOT_ROOT"
+        break
+    else
+        log_verbose "$pref/loader/entries not found…"
+    fi
```

Comment:
Actually this *does* change behaviour, for the better. If we have multiple entries in `$ENTRY_TOKEN_SEARCH`, and `$pref/loader/entries` exists, we would always exit after the first inner loop and ignore the second and later candidates.

Reply:
So… please just adjust the commit message.

Reply:
Indeed. I did not notice that.